homework-04/routes/authentication.js

```diff
@@ -4,36 +4,40 @@ import passport from 'passport'
 import LocalStrategy from 'passport-local'
 import FacebookStrategy from 'passport-facebook'
 import TwitterStrategy from 'passport-twitter'
-import User from '../models/User'
+import dotenv from 'dotenv'
+
+dotenv.config()
 
 const router = express.Router()
 
 const GoogleStrategy = require('passport-google-oauth').OAuth2Strategy
+const User = require('../models').User
 
-const usersDb = User.getUsersDB()
 const secret = process.env.tokenSecret
 
 router.post('/local', (req, res) => {
   if (req.body && req.body.email) {
-    const user = usersDb.find(user => user.email === req.body.email)
-    if (user && user.password === req.body.password) {
-      let data = {
-          user: user
-      }
-      let token = jwt.sign(data, secret, { expiresIn: process.env.tokenExpireTime })
-      let payload = {
-        code: 200,
-        message: 'OK',
-        data: data,
-        token: token
-      }
-      res.status(200).json(payload)
-    } else {
-      res.status(404).json({
-        code: 404,
-        message: 'Not Found'
-      })
-    }
+    User.findOne({where: {email: req.body.email, password: req.body.password}})
```

Comment:
move querying logic to `controllers` directory: this will help you easier migrate to mongodb

Reply:
moved